Functions.Templates/Templates/HttpTrigger-Python/__init__.py

```diff
@@ -1,12 +1,20 @@
 import logging
 
 import azure.functions as func
-
+# Add any dependencies you might want here and in requirements.txt
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
+    """
+    This is the function that will be called when calling your Azure Function.
+    Currently, it is a template for an HTTP Trigger that says hello when calling your URL with a parameter 'name'
+    Try changing the code and run it locally in VS Code by pressing F5!

```

Comment:
One thing to mention here is about the trigger name **HttpTrigger2**, since this is a static template, the HttpTrigger2 may not exist in customer's function. Use something like angle bracket `<HttpTriggerName>` to highlight this part as it may vary.

Reply:
Yeah. This was just an example for the issue but yes, this should be generic example. Even the port should be mention as a variable, with mention of 7071 as default.

Reply:
@ijoosong mind addressing the feedback here and we can try and get this PR closed out :)